packages/framework/src/Component/Elasticsearch/Exception/ElasticsearchIndexException.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\Elasticsearch\Exception;
+
+use Exception;
+
+class ElasticsearchIndexException extends Exception
+{
+    /**
+     * @param string $documentName
+     * @param string $definitionFilename
+     * @return self
+     */
+    public static function invalidJsonInDefinitionFile(string $documentName, string $definitionFilename): self
+    {
+        return new self(sprintf(
+            'Invalid JSON in %s definition file %s',
+            $documentName,
+            $definitionFilename
+        ));
+    }
+
+    /**
+     * @param string $definitionFilename
+     * @return self
+     */
+    public static function cantReadDefinitionFile(string $definitionFilename): self
+    {
+        return new self(sprintf(
+            'Can\'t read definition file at path %s. Please check that file exists and has permissions for reading.',
```

Comment:
_nitpick_ 🤓 
Variable is named `definitionFilename`, but here it's referred as the path. It's confusing to me

Reply:
renamed to `$definitionFilePath` also with other related wrong namings.